kani-compiler/src/kani_middle/points_to/points_to_analysis.rs

```diff
@@ -53,7 +53,7 @@ struct PointsToAnalysis<'a, 'tcx> {
     tcx: TyCtxt<'tcx>,
     /// This will be used in the future to resolve function pointer and vtable calls. Currently, we
     /// can resolve call graph edges just by looking at the terminators and erroring if we can't
-    /// resolve the callee.  
+    /// resolve the callee.
```

Comment:
Whitespace change?

Reply:
This is being enforced by cargo fmt, not a manual change.